core/jvm/src/main/scala/zio/internal/ZScheduler.scala

```diff
@@ -483,19 +486,17 @@ private[zio] object ZScheduler {
 
   private object Locations {
 
-    final class Enabled(initialSize: Int = 64) extends Locations {
+    final class Enabled(sizeHint: Int = 64) extends Locations {
       private[this] val locations = mutable.HashMap.empty[Trace, AtomicLong]
-      locations.sizeHint((initialSize / 0.75d).toInt)
+      locations.sizeHint(sizeHint)
 
       def get(trace: Trace): Long = {
         val v = locations.getOrElse(trace, null)
         if (v eq null) 0L else v.get()
       }
 
       def put(trace: Trace): Long =
-        locations
-          .getOrElseUpdate(trace, new AtomicLong(0L))
-          .getAndIncrement()
+        locations.getOrElseUpdate(trace, new AtomicLong()).getAndIncrement()
```

Comment:
Why removing the explicit `0L` in `new AtomicLong(0L)`? I personally prefer when things are explicit. Does it change anything in terms of performances?

Reply:
Agh just an oversight. I'll add it back